packages/api/app/Models/Technology.js

```diff
@@ -85,20 +86,47 @@ class Technology extends Model {
 		query.with('terms.taxonomy');
 	}
 
+	static async scopePublished(query) {
+		query.where({ status: 'published' });
+	}
+
 	/**
 	 * Query scope to get the technology either by id or slug
 	 *
 	 * @param {object} query The query object.
 	 * @param {number|string} technology The technology id or slug
 	 *
-	 * @returns {object}
 	 */
 	static scopeGetTechnology(query, technology) {
 		if (Number.isInteger(Number(technology))) {
-			return query.where({ id: technology });
+			query.where({ id: technology });
+		} else {
+			query.where({ slug: technology });
+		}
```

Comment:
```suggestion
			return query.where({ id: technology });
		}

	    return query.where({ slug: technology });
```

return early para evitar chains de if/else 👍